### PR TITLE
feat: cowswap honor slippage in signed order

### PR DESCRIPTION
**Pull request by gomesalexandre**
## Description

Reported by CoW directly:

> Are you not subtracting the slippage tolerance from the quote in your code? If not, this leads to effectively 0 slippage being applied which can lead to orders taking a very long time to settle or not settling at all.

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

closes https://github.com/shapeshift/web/issues/6365

> What protocols, transaction types or contract interactions might be affected by this PR?

Low - ensure slippage is deducted in the buy amount in the quote to-be-signed

## Testing

- The minimum buy amount in CoW is correctly slippage-deducted (both with default 50bip and custom slippage tolerance)
- CoW orders fill much faster 🎉 

### Engineering

- ^

### Operations

- ^

## Screenshots (if applicable)

Note, quote with slightly different rate since I tested this after, but just a sanity check while at it to ensure custom slippage UI looks sane:

### CODE REVIEW

**Comment by gomesalexandre** [comment]
* **#6394** <a href="https://app.graphite.dev/github/pr/shapeshift/web/6394?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `develop`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @gomesalexandre and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/lib/swapper/swappers/CowSwapper/endpoints.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { fromAssetId } from '@shapeshiftoss/caip'
-import type { EvmChainId } from '@shapeshiftoss/chain-adapters'
+import { type EvmChainId } from '@shapeshiftoss/chain-adapters'
```
Comment (gomesalexandre) [inline src/lib/swapper/swappers/CowSwapper/endpoints.ts:2]: revert me

Comment (gomesalexandre) [reply src/lib/swapper/swappers/CowSwapper/endpoints.ts]: 2252fa315e

**Review by kaladinlight — APPROVED** [review]
weird api... but does what is says on the box

**Pull request merged**